IntegrationTests/appsettings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ConnectionStrings": {
+    "DefaultConnection": "server=127.0.0.1;port=3306;database=admindashboard;user=root;password=mmyAws@eAF57%tpHhP"
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*"
+}
```

Comment:
Is there a way of reusing the `appsettings.json` file from the AdminDashboard folder?

Reply:
There's probably a way, but it's best to keep them separate on a per-project basis. I foresee these two settings diverge at some point (Integration test's will point to local external APIs, and the AdminDashboard will point to dev, stage, prod)!